### PR TITLE
Map the Travis CI environment variables to similar GitHub Actions equivalents.

### LINKED ISSUE #1862: [GitHub Actions] Map the Travis CI environment variables to similar GitHub Actions equivalents.
### What article on docs.github.com is affected?

- [Migrating from Travis CI to GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions)

### What part(s) of the article would you like to see updated?

- [Default environment variables](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions#default-environment-variables)

### Additional information

Create a table that matches the default Travis CI environment variables with any similar GitHub Actions equivalents. Add a column that describes the behavior of the GitHub Actions environment variable, adding any relevant notes.

### PR DESCRIPTION
**Pull request by martin389**
### Why:

This PR adds the required content for https://github.com/github/docs/issues/1862

### What's being changed:

- Added a new table that maps Travis variables to similar GitHub Actions equivalents.
- Added small edits to environment variable descriptions.

### Changes in this PR

| Article in staging  | Changes |
| --- | --- | 
“[Migrating from Travis CI to GitHub Actions](https://docs-1869--1862-add-travis-ci.herokuapp.com/en/actions/migrating-to-github-actions/migrating-from-travis-ci-to-github-actions#default-environment-variables)” | Added new table that maps Travis variables to GH Actions.

### Check off the following:
- [ ] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by mehdi-miner** [comment]
👍

**Comment by mehdi-miner** [comment]
😎✅

File: content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md [hunk 1]
```diff
@@ -39,6 +39,24 @@ Travis CI lets you set environment variables and share them between stages. Simi
 
 Travis CI and {% data variables.product.prodname_actions %} both include default environment variables that you can use in your YAML files. For {% data variables.product.prodname_actions %}, you can see these listed in "[Default environment variables](/actions/reference/environment-variables#default-environment-variables)."
```
Comment (lm322) [inline content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md:40]: Huy

File: content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md [hunk 2]
```diff
@@ -39,6 +39,24 @@ Travis CI lets you set environment variables and share them between stages. Simi
...
 
+To help you get started, this table maps some of the common Travis CI variables to {% data variables.product.prodname_actions %} variables with similar functionality:
+
```
Comment (lm322) [inline content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md:43]:
```suggestion

```

**Review by lm322 — CHANGES_REQUESTED** [review]

**Review by moneyglitxh — APPROVED** [review]

**Review by stevenans9859 — APPROVED** [review]

**Review by stevenans9859 — APPROVED** [review]

**Review by stevenans9859 — APPROVED** [review]

**Comment by stevenans9859** [comment]
My pleasure, it was a great pleasure to work with you all

File: content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md (hunk 1 above)
Comment (connectionjewelry) [reply content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md:40]: igrating-from-travis-ci-to-github-actions.md
@@ -39,6 +39,24 @@ Travis CI lets you set environment variables and share them between stages. Simi

 Travis CI and {% data variables.product.prodname_actions %} both include default environment variables that you can use in your YAML files. For {% data variables.product.prodname_actions %}, you can see these listed in "[Default environment variables](/actions/reference/environment-variables#default-environment-variables)."

 To help you get started, this table maps some of the co

File: content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md (hunk 2 above)
Comment (connectionjewelry) [reply content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md:43]: igrating-from-travis-ci-to-github-actions.md
@@ -39,6 +39,24 @@ Travis CI lets you set environment variables and share them between stages. Simi

 Travis CI and {% data variables.product.prodname_actions %} both include default environment variables that you can use in your YAML files. For {% data variables.product.prodname_actions %}, you can see these listed in "[Default environment variables](/actions/reference/environment-variables#default-environment-variables)."

 To help you get started, this table maps some of the co

File: content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md (hunk 1 above)
Comment (Ya200d) [reply content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md:40]: 🤚

**Comment by lokera07** [comment]
Gracias

**Review by Mandymoore680 — APPROVED** [review]

**Review by Mandymoore680 — CHANGES_REQUESTED** [review]
Good

**Review by Mandymoore680 — CHANGES_REQUESTED** [review]
Goog

**Comment by janiceilene** [comment]
Sorry @martin389! It looks like our new action inadvertently closed this :upside_down_face:

**Review by dwdjanika — APPROVED** [review]

**Review by lucalves — APPROVED** [review]

**Review by goeltea — COMMENTED** [review]
Semoga lebih baik

**Pull request closed without merging**